Data/Csv/Conversion.hs

```diff
@@ -13,7 +13,7 @@
     UndecidableInstances
     #-}
 
-#if MIN_VERSION_base(4,9,0)
+#if __GLASGOW_HASKELL__ >= 800
```

Comment:
Btw, does enabling those extensions in this module change any exposed API of `cassava`?